checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -1,106 +1,109 @@
-# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
 # ReFrame Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
 import contextlib
+import pathlib
+import sys
+
 import reframe as rfm
+import reframe.utility.sanity as sn
+
+sys.path.append(
+    str(pathlib.Path(__file__).parent.parent.parent.parent / 'mixins')
+)
 
+from extra_launcher_options import ExtraLauncherOptionsMixin
 from hpctestlib.microbenchmarks.mpi.osu import (build_osu_benchmarks,
```

Comment:
can we move to a more recent version of osu ? 
- OSU Micro Benchmarks v5.9 (03/02/2022)
- OSU Micro Benchmarks v7.2 (07/10/2023)